app/v1/controllers/RetrieveSelfEmploymentBISSController.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers
+
+import cats.data.EitherT
+import cats.implicits._
+import javax.inject.{Inject, Singleton}
+import play.api.http.MimeTypes
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import utils.Logging
+import v1.controllers.requestParsers.RetrieveSelfEmploymentBISSRequestDataParser
+import v1.models.errors._
+import v1.models.requestData.RetrieveSelfEmploymentBISSRawData
+import v1.services.{EnrolmentsAuthService, MtdIdLookupService, SelfEmploymentBISSService}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class RetrieveSelfEmploymentBISSController @Inject()(
+                                                      val authService: EnrolmentsAuthService,
+                                                      val lookupService: MtdIdLookupService,
+                                                      requestParser: RetrieveSelfEmploymentBISSRequestDataParser,
+                                                      selfEmploymentBISSService: SelfEmploymentBISSService,
+                                                      cc: ControllerComponents
+                                          )(implicit ec: ExecutionContext)
+  extends AuthorisedController(cc)
+    with BaseController
+    with Logging {
+
+  implicit val endpointLogContext: EndpointLogContext =
+    EndpointLogContext(
+      controllerName = "RetrieveSelfEmploymentBISSController",
+      endpointName = "retrieveBiss"
```

Comment:
Can this be more specific? Otherwise the other endpoint is likely to have the same name.

Reply:
That is the method name inside SelfEmployment...